.travis.yml

```diff
@@ -3,8 +3,10 @@ compiler:
   - clang
   - gcc
 env:
-  - ARCH=i386
-  - ARCH=amd64
+  - ARCH=i386 ENABLE_SHARED_EXECUTABLES=YES
+  - ARCH=amd64 ENABLE_SHARED_EXECUTABLES=YES
+  - ARCH=i386 ENABLE_SHARED_EXECUTABLES=NO
+  - ARCH=amd64 ENABLE_SHARED_EXECUTABLES=NO
```

Comment:
Nope, this is right.  I want to make sure the test cases verify `cork-hash` and friends once each for when they're statically linked and dynamically linked.  (This should give us a 2x2 build matrix — 32-bit vs 64-bit and static vs shared executables.)